setup.cfg

```
@@ -34,6 +34,7 @@ test =
     speclite>=0.11
 all =
     speclite>=0.11
+    colossus
 docs =
     sphinx-astropy
     matplotlib
```

```suggestion
    matplotlib
    colossus
```
This is needed so ReadTheDocs will install colossus before trying to build the documentation